helm/alfresco-content-services/charts/alfresco-content-common/templates/_helpers-image-pull-secrets.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "alfresco-content-services.imagePullSecrets" }}
+{{- if .Values.global.alfrescoRegistryPullSecrets }}
+# only set this secret if a private docker registry variable is defined
+imagePullSecrets:
+    - name: {{ .Values.global.alfrescoRegistryPullSecrets }}
+{{- end }}
+{{- end }}
```

Comment:
alfresco-cotent-common subchart should already be dependency of all the other subcharts, so in the end it's only needed to be defined once (and we don't want to duplicate the same things in so many files anway)